coffee-chats/src/main/java/com/google/step/coffee/entity/ChatRequest.java

```diff
@@ -1,28 +1,33 @@
 package com.google.step.coffee.entity;
 
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
 import java.util.Date;
 import java.util.List;
 
 /** Represents a request made by a user for a chat on given topics. */
 public class ChatRequest {
```

Comment:
I'm probably late for this, but why not AutoValue?

Reply:
Not currently used elsewhere outside of scheduling and building datastore entities. In the future I may need to return ChatRequests to user, at which point I shall migrate over to using AutoValue.